templates/saga/package.json

```diff
@@ -1,43 +1,43 @@
 {
-  "name": "create-next-app",
+  "name": "my-saga",
```

Comment:
I think `create-next-app` is more generic than `my-saga`. What do you think if we revert back to the origin?

Reply:
Ya sure I agree. I think I miss committed this line.

Reply:
So can u commit again?

Reply:
done